blocks/rich-text/index.js

```diff
@@ -283,11 +283,19 @@ export default class RichText extends Component {
 	 * @param {PasteEvent} event The paste event as triggered by tinyMCE.
 	 */
 	onPaste( event ) {
-		const dataTransfer = event.clipboardData || event.dataTransfer || this.editor.getDoc().dataTransfer;
-		const { items = [], files = [] } = dataTransfer;
+		const dataTransfer =
+			event.clipboardData ||
+			event.dataTransfer ||
+			this.editor.getDoc().dataTransfer ||
+			{ getData: () => '' };
```

Comment:
Interesting :)

Reply:
I noticed the code previously did not fall back to an object, so if this ends up `undefined` the following line would error. Also no need for further `dataTransfer` checks then. I don't even know if this is a real scenario, but better safe than sorry.

Reply:
Sounds like a good inline comment.

Reply:
Done.